pkgs/applications/misc/mepo/default.nix

```diff
@@ -46,12 +58,26 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
+  postInstall = ''
+    wrapProgram $out/bin/mepo_dl.sh\
+      --prefix PATH : ${lib.makeBinPath [ jq ncurses ]}
+    wrapProgram $out/bin/mepo_ui_helper_menu.sh\
+      --prefix PATH : ${lib.makeBinPath menuInputs}
+    for script in $(grep -l jq out/bin/mepo_ui_menu_*.sh); do
+      wrapProgram $script --prefix PATH : $out/bin:${lib.makeBinPath [ jq ]}
+    done
+    for prog in $out/bin/mepo*; do
+      if [ ! -f $out/bin/.$(basename $prog)-wrapped ]; then
+        wrapProgram $prog --prefix PATH : $out/bin
```

Comment:
```suggestion
        wrapProgram $prog --suffix PATH : $out/bin
```

Reply:
Should `$out/bin` be preferred over what's already in PATH?